frontend/packages/kubevirt-plugin/src/statuses/vm/vm.ts

```diff
@@ -249,6 +256,29 @@ export const getVMStatus = ({
   );
 };
 
+export const getVMIStatus = ({
```

Comment:
IMO we should refactor the VM statuses. Nevertheless could we separate VMI status and do it in a followup?

Reply:
+1 for refactoring, leaving as is for now.

Reply:
merged the vm and vmi get status methods